mev-boost-rs/src/relay_mux.rs

```diff
@@ -17,6 +17,8 @@ use thiserror::Error;
 // See note in the `mev-relay-rs::Relay` about this constant.
 // TODO likely drop this feature...
 const PROPOSAL_TOLERANCE_DELAY: Slot = 1;
+// Wait for relays to fetch bids until time out in seconds.
+const FETCH_BEST_BID_TIME_OUT: u64 = 3;
```

Comment:
my personal stylistic take is that `usize` is the correct type for "cardinal" types, like "how many seconds we want"

so then this type would be updated in lieu of `u64`

Reply:
I used `u64` because that's the type tokio wants to convert to `Duration`. What do you think?

Reply:
yeah that's fine then